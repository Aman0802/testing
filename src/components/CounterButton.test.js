import { shallow } from "enzyme";
import React from "react";
import CounterButton from "./CounterButton";

it("expect CounterButton component to rendr", () => {
  const mockColor = "red;";
  expect(shallow(<CounterButton red={mockColor} />)).toMatchSnapshot();
});

it("correctly increments the counter", () => {
  const mockColor = "red;";
  const wrapper = shallow(<CounterButton red={mockColor} />);
  wrapper.find('[id="counter"]').simulate("click");
  expect(wrapper.state()).toEqual({ count: 1 });
});
