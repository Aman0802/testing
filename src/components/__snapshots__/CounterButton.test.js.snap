// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect CounterButton component to rendr 1`] = `ShallowWrapper {}`;
